### ISSUE TITLE
F4SE not working with MO2

### ISSUE THREAD

**Issue by Matth792** [TanninOne/modorganizer#1131]
Hi all, my game recently just started crashing on startup (before any intro slideshow it gives the "Fallout 4 has stopped working" error) when running F4SE through MO2. I've been fiddling with a bunch of different things to try and fix this but I'm confused since the original crashing started occurring without me having done anything different. 

If I try running F4SE outisde of MO2, it will still work and start up the game, but does not recognize my mods installed. I tried the solutions mentioned in [https://github.com/TanninOne/modorganizer/issues/1086](url) and [https://github.com/TanninOne/modorganizer/issues/649](url) but none of them seem to work (ex: uninstalling ENB). I'm currently using version 2.0.7. Does anyone have any suggestions on how to fix/debug this?

**Comment by JPointman11** [comment]
Have you played the game with your selected mods before the crashing started? If not, it is some of your mods that causes your crashes.

**Comment by Matth792** [comment]
Yeah, pretty sure it was fine. I tried deactivating every single mod but it still won't even make it to the start screen.
